Dockerfile

```diff
@@ -10,6 +10,16 @@ RUN set -ex; \
     echo 'no-cache-dir = false' >> /etc/pip.conf; \
     echo "extra-index-url = https://$codename.wheelhouse.praekelt.org/simple" >> /etc/pip.conf
 
+# HACK: Make sure a Python is available as `python`. Only necessary for PyPy.
+RUN set -ex; \
+    if ! command -v python > /dev/null; then \
+        cd /usr/local/bin; \
+        for py in pypy3 pypy python3 python2; do \
+            [ -x "$py" ] && ln -s "$py" python || true; \
+        done; \
```

Comment:
Will this break if we have more than one of those installed? I suppose it doesn't really matter if we're only ever expecting to have one.

Reply:
Yes, it will break with `ln: python: File exists`, but that should get caught as an error in travis, since it happens during the build. So we won't get any runtime errors with this.